backend/core/taps/ripe_ris.py

```diff
@@ -144,6 +149,16 @@ def parse_ripe_ris(connection, prefixes_file, hosts):
                         ):
                             norm_ris_msgs = normalize_ripe_ris(msg, prefix_tree)
                             for norm_ris_msg in norm_ris_msgs:
+                                redis.set(
```

Comment:
We should add a `timeout` on `events = requests.get` (line 132)
Not having received an update for /0 and ::0 for 10 seconds should timeout the connection and reconnect

Reply:
so you mean
```
events = requests.get(
                    "https://ris-live.ripe.net/v1/stream/?format=json&client=artemis-{}".format(
                        ris_suffix
                    ),
                    stream=True,
                    timeout=10
                )
```
? Do we need any special management for the timeout, could you check what will happen if this happens? I assume it will go to the iterator and then while True?

Reply:
done in code, just check if we need anything else for special management (I think we already have the while True loop that will kick everything back up).